ami/data.py

```diff
@@ -144,7 +146,8 @@ class Message:
     mtype: MsgTypes
     identity: int
     payload: dict
-    timestamp: int = 0
+    timestamp: int = 0  # typically raw data source (LCLS) timestamp
+    unix_ts: int = 0  # timestamp converted to unix_ts
```

Comment:
Should this be float?

Reply:
You're probably right, I'll double check.